services/cd-service/pkg/repository/repository.go

```diff
@@ -1447,6 +1448,13 @@ func (s *State) GetApplicationRelease(application string, version uint64) (*Rele
 	} else {
 		release.SourceMessage = string(cnt)
 	}
+	if cnt, err := readFile(s.Filesystem, s.Filesystem.Join(base, "display_version")); err != nil {
```

Comment:
that's an unsual abbreviation for "content", please rename.
E.g.
```suggestion
	if displayVersion, err := readFile(s.Filesystem, s.Filesystem.Join(base, "display_version")); err != nil {
```